### PR TITLE
fix: fix the bug in upgrade

### PR DESCRIPTION
**Pull request by likyh**
### Summary
fix the bugs in upgrade:
1. a typo when reading name
2. scope -> scopes
3. pass correct scopeId

### CODE REVIEW

File: config-ui/src/error/components/bp-upgrade/use-bp-upgrade.ts [hunk 1]
```diff
@@ -104,25 +104,25 @@ export const useBPUpgrade = ({ id, onResetError }: UseBPUpgradeProps) => {
...
-    const res = await API.updateDataScope(plugin, connectionId, getScopeId(plugin, scopeDetail), {
+    await API.updateDataScope(plugin, connectionId, getScopeId(plugin, scopeDetail), {
       ...scopeDetail,
-      transformationRuleId: transfromationRule?.id,
+      transformationRuleId: transformationRule?.id,
     });
 
     return {
-      id: res.id,
+      id: getScopeId(plugin, scopeDetail).toString(),
```
Comment (mintsweet) [inline config-ui/src/error/components/bp-upgrade/use-bp-upgrade.ts:113]: `${getScopeId(plugin, scopeDetail)}` will better.

File: config-ui/src/error/components/bp-upgrade/use-bp-upgrade.ts [hunk 2]
```diff
@@ -104,25 +104,25 @@ export const useBPUpgrade = ({ id, onResetError }: UseBPUpgradeProps) => {
...
       entities: scope.entities,
     };
   };
 
   const upgradeConnection = async (connection: any) => {
     const { plugin, connectionId } = connection;
 
-    const scope = await Promise.all(connection.scope.map((sc: any) => upgradeScope(plugin, connectionId, sc)));
```
Comment (mintsweet) [inline config-ui/src/error/components/bp-upgrade/use-bp-upgrade.ts:121]: Do not modify this name, because scopes means glasses.

**Review by mintsweet — APPROVED** [review]
LGTM.

**Review by warren830 — APPROVED** [review]
@mintsweet already reviewed

**Pull request merged**